src/m2kcalibration_impl.hpp

```diff
@@ -74,6 +74,10 @@ class M2kCalibrationImpl : public M2kCalibration
 	void setDacOffset(unsigned int chn, int offset);
 	void setAdcOffset(unsigned int chn, int offset);
 	void setAdcGain(unsigned int chn, double gain);
+
+	bool getAdcCalibrated() const;
```

Comment:
this methods can be declared as `override`

Reply:
fixed